src/tests/ftest/osa/online_extend.py

```diff
@@ -142,7 +142,10 @@ def run_online_extend_test(self, num_pool, racer=False,
             display_string = "Pool{} space at the End".format(val)
             self.pool = pool[val]
             self.pool.display_pool_daos_space(display_string)
-            self.run_ior_thread("Read", oclass, test_seq)
+            if app_name == "ior":
+                self.run_ior_thread("Read", oclass, test_seq)
+            else:
+                break
```

Comment:
Repeat of comment from https://github.com/daos-stack/daos/pull/10645#discussion_r1003553847
Why `break`? Should we just skip this `for` loop altogether if `app_name != "ior"`?